src/planscape/forsys/parse_forsys_output.py

```diff
@@ -0,0 +1,188 @@
+import rpy2
+
+import numpy as np
+
+
+class Scenario():
+    # Priority weights for the scenario.
```

Comment:
Should these be instance variables, not class variables?  See https://medium.com/python-features/class-vs-instance-variables-8d452e9abcbd

Reply:
Could you use the TypedDict pattern here from //Planscape/src/planscape/base/condition_types.py ?  It would look like

class RankedProject(TypedDict):
  id: int
  weighted_priority_scores: dict
  total_score: int
  rank: int

class Scenario(TypedDict):
  priority_weights: dict
  ranked_projects: list[RankedProject]
  cumulative_ranked_project_area: list[float]
  cumulative_ranked_project_cost: list[float]

Reply:
oh! I see. yea: they should definitely be instance variables. Thanks.
I see why "self.scenarios = {}" was necessary in __init__ in this iteration and how I can remove it now.

Reply:
done - updated.